src/css/colors.css

```diff
@@ -54,7 +54,7 @@
   --link-background-light: rgba(255, 255, 255, 0.7);
 
   --feedback-yellow-warning-background: #fbeaad;
-
+  --feedback-yellow-warning-background-light: #fdf5d6;
```

Comment:
..virkelig? er det en anden gul her?

Reply:
ja, jeg har lavet lidt akrobatik med overlay for at få det til at passe i materialCard... så den bruger bare --feedback-yellow-warning-background/--warning-light, burde nok havde tilføjet det nye gul den gang...